cli/cmd/sync.go

```diff
@@ -33,16 +34,22 @@ func NewCmdSync() *cobra.Command {
 	return cmd
 }
 
+func getPluralDirectories(count int) string {
+	if count == 1 {
+		return "directory"
+	}
+
+	return "directories"
+}
+
 func sync(cmd *cobra.Command, args []string) error {
 	ctx := cmd.Context()
-	directory := "."
-	if len(args) > 0 {
-		directory = args[0]
-	}
-	fmt.Println("Loading specs from directory: ", directory)
+
+	directoryPlural := getPluralDirectories(len(args))
+	fmt.Printf("Loading specs from %s: %s\n", directoryPlural, strings.Join(args, ", "))
```

Comment:
Btw, since we're here... there is another issue with this log line: if the config being read is a file, it still prints "directory". Sorry to pile onto this PR, but would you mind fixing that too? 😄